src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs

```diff
@@ -7356,6 +7356,13 @@ protected internal void ImportModuleMembers(PSModuleInfo sourceModule, string pr
                 message = StringUtil.Format(Modules.ImportingFunction, prefixedName);
                 cmdlet.WriteVerbose(message);
             }
+            else
+            {
+                string manifestFilename = Path.GetFileName(cmdlet.Context.PreviousModuleProcessed);
+                string moduleFileName = Path.GetFileName(sourceModule.Path);
```

Comment:
You should just search replace `FileName` with `Filename`

Reply:
That's my bad :(
Having said that pretty much everywhere else in that file considers filename to be two words and capitalizes it as such.
Ex : GetFileName(), resolvedFileName, etc